corehq/ex-submodules/couchexport/tests/test_writers.py

```diff
@@ -44,19 +45,31 @@ def test_zipped_export_writer_unicode(self):
         mock_zip_file = self.MockZipFile.return_value
         self.writer.table_names = {0: 'ひらがな'}
         self.writer._write_final_result()
-        mock_zip_file.write.assert_called_with(
-            'tmp',
-            os.path.join(self.writer.archive_basepath, 'ひらがな.csv'.encode('utf-8'))
-        )
+        if six.PY3:
+            mock_zip_file.write.assert_called_with(
+                'tmp',
+                os.path.join(self.writer.archive_basepath, 'ひらがな.csv')
+            )
+        else:
+            mock_zip_file.write.assert_called_with(
+                'tmp',
+                os.path.join(self.writer.archive_basepath, 'ひらがな.csv'.encode('utf-8'))
+            )
```

Comment:
nit: minimizing the code in the `if` block would make it a little easier to quickly see what's going on here
```py
filename = 'ひらがな.csv'
if six.PY2:
    filename = filename.encode('utf-8')
mock_zip_file.write.assert_called_with(
    'tmp',
    os.path.join(self.writer.archive_basepath, filename)
)
```